sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -2046,7 +2047,7 @@ def PatchItem(
         self._UpdateSessionIfRequired(headers, result, last_response_headers)
         if response_hook:
             response_hook(last_response_headers, result)
-        return result
+        return CosmosDictResponse(original_dict=result, response_headers=last_response_headers)
```

Comment:
curious: does this counted as breaking change?

Reply:
thankfully it does not since the behavior remains the same :) for any item response, you can still directly access any information in it (result['id']), and since the object inherits the Dict/List class it is still considered a dict/list in validations

Reply:
update on this, seems like it might be since we might have to update the typehint docs on all our public api return types - @annatisch will let us know 😄